dts/bindings/sensor/st,lis2dh-common.yaml

```diff
@@ -20,8 +19,25 @@ properties:
 
     anym-on-int1:
       type: boolean
-      required: false
       description: |
         Indicates that the device driver should use interrupt 1
         for any movement. This is for boards that only have one
         interrupt line connected from the sensor to the processor.
+
+    anym-no-latch:
+      type: boolean
+      description: |
+        Disable the latching of interrupts for any movement.
+
+    anym-mode:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Select the interrupt mode for any movement. The default of 0
+        is the power-on-reset value.
+      enum:
+        - 0 # OR combination of interrupt events
+        - 1 # 6D movement recognition
+        - 2 # AND combination of interrupt events
+        - 3 # 6D position recognition
```

Comment:
Please make a habit of including useful comments in the description instead of the YAML file, so they show up in the bindings index: https://docs.zephyrproject.org/latest/build/dts/api/bindings.html